** IMPORTANT **
Ensure that postfix-files is *NOT* marked as a configuration file as this is
used to install the files in their correct locations.  Marking it as such will
mean that RPM may not overwrite the current file with the *needed* new file
location list.
--- postfix-2.0.0-20021223/conf/postfix-files.orig	2002-12-22 07:23:42.000000000 +0900
+++ postfix-2.0.0-20021223/conf/postfix-files	2002-12-31 18:51:28.000000000 +0900
@@ -102,8 +102,8 @@
 $config_directory/virtual:f:root:-:644:p
 $config_directory/postfix-script:f:root:-:755
 $config_directory/post-install:f:root:-:755
-$manpage_directory/man1/mailq.1:f:root:-:644
-$manpage_directory/man1/newaliases.1:f:root:-:644
+$manpage_directory/man1/mailq.postfix.1:f:root:-:644
+$manpage_directory/man1/newaliases.postfix.1:f:root:-:644
 $manpage_directory/man1/postalias.1:f:root:-:644
 $manpage_directory/man1/postcat.1:f:root:-:644
 $manpage_directory/man1/postconf.1:f:root:-:644
@@ -115,9 +115,9 @@
 $manpage_directory/man1/postmap.1:f:root:-:644
 $manpage_directory/man1/postqueue.1:f:root:-:644
 $manpage_directory/man1/postsuper.1:f:root:-:644
-$manpage_directory/man1/sendmail.1:f:root:-:644
+$manpage_directory/man1/sendmail.postfix.1:f:root:-:644
 $manpage_directory/man5/access.5:f:root:-:644
-$manpage_directory/man5/aliases.5:f:root:-:644
+$manpage_directory/man5/aliases.postfix.5:f:root:-:644
 $manpage_directory/man5/canonical.5:f:root:-:644
 $manpage_directory/man5/pcre_table.5:f:root:-:644
 $manpage_directory/man5/regexp_table.5:f:root:-:644
@@ -193,6 +193,7 @@
 $readme_directory/RELEASE_NOTES:f:root:-:644
 $readme_directory/RESTRICTION_CLASS_README:f:root:-:644
 $readme_directory/SASL_README:f:root:-:644
+$readme_directory/SENDER_VERIFICATION_README:f:root:-:644
 $readme_directory/ULTRIX_README:f:root:-:644
 $readme_directory/UUCP_README:f:root:-:644
 $readme_directory/VERP_README:f:root:-:644
